app/services/proofing/mock/address_mock_client.rb

```diff
@@ -3,9 +3,15 @@ module Mock
     class AddressMockClient < Proofing::Base
       vendor_name 'AddressMock'
 
-      required_attributes :phone
+      required_attributes :uuid,
```

Comment:
my only worry with this is some flow that doesn't adequately set the uuid but it makes sense that if the other proofers have it mandatory it logically should be there always because address and aamva come after instant verify.  with hybrid flow it's effective_user so there is a little magic but i believe we are ok :-)

Reply:
This was just a true-up to match the PhoneFinder proofer, if it was optional there, it would be optional here